_posts/new.md

```diff
@@ -0,0 +1,6 @@
+---
+layout:slide
```

Comment:
[ ] Turn on Github Pages
[ ] Outline my portfolio
[ ] Introduce myself to the world